geode-core/src/main/java/org/apache/geode/cache/RegionAttributes.java

```diff
@@ -71,6 +71,13 @@
 
   Class<V> getValueConstraint();
 
+  /**
+   * Returns true if configured detect read conflict mode
```

Comment:
Include "transaction" in this line in someway. Perhaps: "Returns true if this region is configured to detect read conflicts when a transaction is committed."